plotIt_hh/listHistos.py

```diff
@@ -54,6 +74,10 @@
 for key in keys:
     if key.GetName() not in alreadyIn and not "__" in key.GetName():
 
+        # Keep only histograms
+        if not key.ReadObj().InheritsFrom("TH1"):
```

Comment:
If you do this you can remove https://github.com/cp3-llbb/HHTools/pull/43/files#diff-19e06715e593fe9eea6306b25e9af506R88 ;)